python/ray/experimental/serve/api.py

```diff
@@ -216,7 +308,7 @@ def scale(backend_tag, num_replicas):
     """
     assert backend_tag in global_state.backend_table.list_backends(), (
         "Backend {} is not registered.".format(backend_tag))
-    assert num_replicas > 0, "Number of replicas must be greater than 1."
+    assert num_replicas >= 0, "Number of replicas must be greater than 1."
```

Comment:
is this intentional? if so let's change the error message to say "greater than 0"